wifi-subsys/components/tools/test/test_cbor.c

```diff
@@ -117,5 +138,24 @@ TEST_CASE("Decode the int value that belongs to key_element[2]", "cbor")
     }
 
     val_received = 0;
+}
+
+TEST_CASE("Decode the uint8_t values that belongs to key_element[3]", "cbor") {
+    array_out = malloc(sizeof(len));
+    esp_err_t err = decode((uint8_t *)cbor_code, key_element[3], array_out, len);
 
+    if (err != ESP_OK) {
+        ESP_LOGE("CBOR", "Decode failed");
+        ESP_LOGD(__func__, "Reason %s, No(%d)", esp_err_to_name(err), err);
```

Comment:
```suggestion
        ESP_LOGE(TAG, "Reason %s, No(%d)", esp_err_to_name(err), err);
```
TAG can be some kind of identifier such as "TEST",  "CBOR_TEST" or similar.

Don't use debug logging in this PR